### DISCUSSION TITLE
Worst stats: Do we display them?

### DISCUSSION THREAD

**Discussion by hburn7** [osu-tournament-rating/otr-web#186] [Q&A] [answered]
There seems to be a perfect rift between whether to display the 'worst stats' publicly, such as a player's worst tournament performances. On one hand, we don't want to introduce metrics that make players actively feel bad about themselves. On another hand, some points have been made regarding how it is useful for things like team building. Everyone who plays 6+ tournaments will have at least one "worst" performance on this chart.

About 50% voted for "Yes, display the worst stats publicly" and another ~50% voted for either "Yes, display the stats privately (e.g. on one's own dashboard)" with a small minority voting to not include these stats at all.

 I'd like to have a more in-depth discussion on this topic. Input from players is especially valuable. Would it feel bad to have your worst performances on your profile, or would it not feel bad & be a useful metric for yourself and possibly for team building?

**Comment by Niviferum** [comment]
It’s a good metric to show one’s consistency. Just an idea though, can you guys display a percentage of how often scores similar to those happen ? It helps for team building imo, so captains can know if one is more often at their peak or at their worst. It also shows the nerves of someone

**Reply by hburn7** [reply]
We can display quite a lot of things, but we have different priorities for now. Eventually we'll have a feature request forum.

**Reply by Niviferum** [reply]
Also, I forgot about the aspect of « it makes someone feel bad ». Imo people who indeed feel bad about their worst perf are simply not able to move on with their past. We can’t always be on top, and people have to be aware of what could happen in case of a bad day. Also it has to make sure that 0 points scores are not counted in because then people could try to hide their worst scores with a simple throw and a 0 value

**Comment by Userlog2** [comment]
There could be a time limit that can be toggled by the individual user on how far back to show worst performances, for example:
- 18 months (minimum)
- 24 months
- 30 months
- 36 months
- [...]
- No limit

This would allow for effective team-building while limiting a user's discouragement, since no one really needs to see a choked tournament 3 years ago.

**Comment by germanysbutcher** [comment]
i think displaying somebodys worst stats is extremly helpful to work on your weaknesses and realise the stuff that you wouldnt normally look at. you can always learn more from getting 0-5'ed than 5-0'ing someone.

**Comment by Nopekjk** [comment, marked as answer]
The reason I'm against showing them publicly (and even privately too) is, it's far away from what we 'are', but rather what we 'were'. In other words, it's far away from what we can 'improve', but rather a 'stain'.

1. In aspect of skillset, what we call weak (or worst) point is, what we can work on. But in the other side, the worst performed tournament is already ended, and leaves no room for improvement. Also, as someone who witnessed a lot of tournament players who mourn and regret about their old under-performance - even from several years ago - I think showing the worst stats can be detrimental especially for those players who are very sensitive of their performance & result.

2. Generally when we talk about team building, although some players check their recent performance and assess their current 'form', mostly the process of recruiting players are based on skillset: such as, "we are lacking speed players, so let's search for players who have high speed rating (on skill issue bot, or anything)". Plus, If we want the metric that evaluates players' performance that compressed into a number, I would go with 'recent performance' approach: This way, the team builders can 'invest' the players with high expectation, and the players can see how they are doing more efficiently.

The best performance section looks great: good memories are what makes the player motivated and thrilled, it's the similar logic with why tournament players are looking forward profile badges. In contrary, worst performance is like a scar: it might work as the reminder and review notes, but the misses can be never corrected after.

**Reply by hburn7** [reply]
I think this is a very helpful insight. You're right, it does feel like a stain that can't be removed. I think replacing this item with 'recent performances' could be more insightful without leaving that 'stain feeling' (since any bad performance would move away from your profile after 5 tournaments). What do you think about this?

**Reply by Nopekjk** [reply]
I think 'recent performance' is very reliable and reasonable way, plus it's what most of the tournament-based wiki (something like HLTV) uses too. But I would say - as [Userlog2 said](https://github.com/osu-tournament-rating/otr-web/discussions/186#discussioncomment-9446906) - it would be nicer to not display the tournament older than certain period of times, somewhere around 12-18 months.

**Reply by Niviferum** [reply]
How about the old tourneys can only be seen by the owner of the account only ? Just so you can see progress you know what I mean, and caps can simply check the recent perfs

**Reply by Userlog2** [reply]
Agree with the above. I also think the user should be able to hide/suppress their worst tournament performances from themselves, as [MintLeaf said,](https://github.com/osu-tournament-rating/otr-web/discussions/186#discussioncomment-9448428) and that the shown history limit I mentioned should be separate for the public and the user themselves.

**Reply by IMaxwellI** [reply]
Basically came here to comment the same. Worst performance is something you will never be able to improve upon, and thus doesn't really make sense. Also, for someone using that metric to correctfully judge what they read, they'd have to know the tournament that the worst performance was in. E.g. we will have a lot of rank-restricted tournaments in there and the worst performance for a 5/6 digit might be their shot at an open rank tournament or a tournament of higher rank range, meaning the information is also not really valuable. Their worst performance might be their first tournament ever, or one far above their skill level.

Recent performance is definitely the most important and most useful metric that o!TR could display.

**Reply by hburn7** [reply]
I will replace worst tournament performances with recent tournament performances. Any other 'worst' stats will likely get changed to 'recent' stats instead.

**Comment by liawua** [comment]
i don't really see how this information is useful at all, sometimes things happen and you play really badly, you might have just got home and are sightreading the pool or any number of other things. without the context this stat means nothing. also, this stat will only go down as you play more matches, so a bad "worst" match could actually just mean you've played more matches and so have a larger sample size.

it could be worth thinking about showing a "average bad match" which could just be something as simple as the lower quartile match, but as it stands i think showing the worst match is pointless.

**Comment by m1ntleaf** [comment]
I think that there should at least be a setting or spoiler box if we were to show worst performances. It could be helpful, but for many players it would just end up being discouraging seeing it on their profile. I also think that the time filter is a good idea.
